src/scripts/crowdin-import.js

```diff
@@ -0,0 +1,369 @@
+const { copyFileSync, existsSync, mkdirSync, readdirSync } = require("fs")
+const { resolve, join } = require("path")
+
+/**
+ * 0. Console summary flags
+ */
+const VERBOSE = false
+const FULL_BUCKET_NAME_SUMMARY = false
+
+/**
+ * 1. Copy languages folder from Crowdin export to ./.crowdin, or enter override
+ * to point to local unzipped download folder
+ * ie: /Home/alice/Downloads/Ethereum.org Translations (translations)/
+ */
+const CROWDIN_ROOT_OVERRIDE = ""
+
+/**
+ * 2. Select languages and buckets to import by adding the number of the
+ * corresponding content bucket to the `buckets` array
+ * ie. `buckets: [1, 10],` would import the "Homepage" and "Learn" buckets
+ */
+const SELECTION = [
```

Comment:
Nice to have. Migrate this to TS and run it using `ts-node`.

With TS we could could read all the langs from our `src/utils/languages.ts` and keep this in sync. Just an idea:

```ts
const baseBucketsPerLang = supportedLanguages.reduce((res, lang) => {
  return {
    [lang]: []
  }
}, {})

const userSelection = {
  // overwrite w user selection
  en: [1, 10]
}

// or transform this to an array as you have
const bucketsPerLang = {
  ...baseBucketsPerLang,
  ...userSelection
}
```

Reply:
Just to note here, the `SELECTION` object does not need to contain all the languages; the array is hard-coded this way as a template, to make it easier to just enter the buckets, instead of having to type out the object structure at all.  Building the object using code doesn't really add anything (empty langs will just be ignored).

With our import flow, I don't see us using any base buckets since rarely if ever are we importing an updated bucket for every language at the same time.

That being said, I do like the simplified `userSelection` object. Had gone with an array iteration approach, but think this looks cleaner.

Reply:
Perfect. Make sense.